app/javascript/javascript/map.js

```diff
@@ -1,65 +1,50 @@
 import _ from "underscore";
-import { buildCustomMarker } from "./custom_marker";
+import { buildCustomMarker } from "./customMarker";
 import MarkerClusterer from "@google/markerclustererplus";
-import {
-  fixedClusterStyles,
-  pulsingMarkerStyles
-} from "../../../../javascript/theme";
+import { fixedClusterStyles, pulsingMarkerStyles } from "./theme";
 import {
   setHasChangedProgrammatically,
   getHasChangedProgrammatically,
   onMapInit
-} from "../../../../javascript/mapsUtils";
-
-export const map = (
-  params,
-  $cookieStore,
-  $rootScope,
-  digester,
-  rectangles,
-  googleMaps,
-  heat,
-  $window
-) => {
-  const TIMEOUT_DELAY = process.env.NODE_ENV === "test" ? 0 : 1000;
+} from "./mapsUtils";
+import heat from "./heat";
+import rectangles from "./rectangles";
+import googleMaps from "./googleMaps";
+import pubsub from "./pubsub";
+import * as Cookies from "./cookies";
+import params from "./params2";
+
+export default (() => {
+  const TIMEOUT_DELAY = 1000;
   setHasChangedProgrammatically(false);
-  $window.__traceMarkers = [];
-  const elmApp = $window.__elmApp;
+  window.__traceMarkers = [];
+  const elmApp = window.__elmApp;
 
   var Map = function() {};
 
   Map.prototype = {
     init: function(element, options) {
       this.mapObj = googleMaps.init(element, options);
-      if (process.env.NODE_ENV !== "test") onMapInit();
```

Comment:
I'm experiencing a little bout of joy every time I see one of those

Reply:
Hehe, as they say "you've got to crack a few eggs to make an omelette". I think that was a good trick while removing Angular!